### ISSUE TITLE
Agenda for Sept. 21, 2023

### ISSUE THREAD

**Issue by nairnandu** [web-platform-tests/interop#453]
Here is the proposed agenda for Sept 21, 2023

- Finalize the proposal selection process #390 
  - Current list of open comments: https://github.com/web-platform-tests/interop/pull/390#issuecomment-1725455364  
- Review open [test change proposals](https://github.com/web-platform-tests/interop/issues?q=is%3Aissue+is%3Aopen+label%3Atest-change-proposal)

Meeting notes

Attendees - @meyerweb, @nairnandu, @zcorpan, @foolip, @chrishtr, @jgraham, @dandclark, @gsnedders, @boazsender, @jensimmons, @nt1m

Notes:

*   Finalize the proposal selection process Add a proposal for the Interop 2024 process #390
    *   Current list of open comments: Add a proposal for the Interop 2024 process #390 (comment)
    *   jensimmons: votes/vetoes should be replaced with positions
    *   Process for carrying over focus areas from 2023 into 2024. @jgraham suggested filing an issue for each focus area and discussing there if the remaining test failures are important or not. I think we should do that, and think it'll be easy-ish to agree about the carry over afterwards.
        *   foolip: ultimately we should end up with a list of areas that we jointly agree to carry forward
        *   jensimmons: crux is if we should have a process for carryover with a default option of consensus to remove
        *   jgraham: happy to have a separate process, but leaning towards carrying forward as the default option. Individual orgs can propose to exclude something
        *   foolip: agree that we should have consensus on the areas that impact the metric. Suggest a meeting to review the current areas and make a decision
        *   jensimmons: agree to open issues for each area and then we can try to get consensus ahead of the meeting
        *   nt1m: I would propose that we give higher priority for carryover over new proposals
        *   foolip: I think we can enumerate the areas first
        *   jgraham: my skepticism here would be that we won't have enough spots for new proposals. On the flip side, the carryover work might be small enough.
        *   boazsender: I think it’s okay. Its fair to accept that Interop takes a long time and its reasonable to say that something needs 18 months instead of an arbitrary period of 1 year.
        *   jensimmons: agree. One of the questions for carryover discussion would be to also consider the whole amount of work involved.
        *   foolip: +1 to staying flexible on the number of areas and considering the total effort involved
        *   jensimmons: the Safari score is broken and it needs to be fixed before we can do that evaluation.
        *   chrishtr: what would be the criteria for evaluation?
        *   jensimmons: the remaining failures and its impact on web developers
        *   zcorpan: if we identify low-impact tests, should we just remove the tests from the focus area?
        *   foolip: there would be some gray areas and its possible we might uncover some areas like that
        *   jgraham: we might also uncover areas where we could add more tests to the area. This would be a good place to discuss those as well.
        *   boazsender: same process for investigation areas?
        *   foolip: finish by default
        *   gsnedders: the scope would have to be meaningfully different for it to be carried over
        *   jensimmons: maybe each group can open its own issue and we can review that
        *   Next step: foolip@ will create a label and an issue for each focus area
        *   Next step: boazsender@ will create an issue for accessibility investigation area
    *   In round 2, remove or define the limit N on the number of proposals each organization can pick. I suggest we remove it and go with "ask people to pick a number of proposals that they feel reflects a reasonable size for Interop 2024" as suggested by @jgraham. Also allow each org to group proposals as they think would make sense in this round.
        *   jensimmons: do we want it to be smaller or bigger?
        *   jgraham: grouping of proposals makes it difficult to pick an ‘N’. 
        *   gsnedders: when we do the grouping also plays a part in this process
        *   jensimmons: what if we make that a high # (like 50)?
        *   dandclark: striking the right balance between a large number and a very restrictive one makes sense to me as well
        *   foolip: each org picking a max of 50 works as well
        *   jgraham: agree on 50
        *   Consensus to make N as 50
    *   Define what's member-confidential, leaning on Create the Interop Team Charter #102. @gsnedders @jgraham can either of you suggest the wording you'd prefer?
        *   jensimmons: a more clear definition of what is confidential and not would be needed here.
        *   chrishtr: any document that does not have public/confidential specified will be public by default. Also any process that need to be landed must have consensus
        *   chrishtr: the wording can be that “process documents are adopted/published by consensus”
        *   Next step: jensimmons: we will try to wordsmith this and come back with a recommendation
    *   Fix the wording for vetos
        *   Next step: PR390 to be updated to fix the wordings around vetos

**Comment by jensimmons** [comment]
We discussed today making N = 50. 

I tossed out that number, without considering the overlap that will make the total grow.

As I work through the proposals, it seems 50 is far too high. We do want to narrow the field a bit. 

Let's change N to be 35.

**Comment by nairnandu** [comment]
adding this to the agenda for Sept 28th
